### PR TITLE
Add summary page for member

### LINKED ISSUE #37: [Improvement] [FE] Account Popover Summary Pages
Currently, our account popover looks like this.
Image

1. Add the "My Summary" button. You can keep the Weeks thing.
Image

2. If the user is admin, route them to RA Summary page. This is the route /ra-summary
Image

3. If the user is NOT admin, route them to this page. Create the page for it as well.
Image

   - For the scheduled sessions, create functionality to display what month we are in (i.e August, July, ...) but no need to do the showing of the time or dates (this will be implemented later)
   - Just do frontend of the Tasks, Notes, and Hours section. 
   - Create buttons for the Weekly Check-in, T2W, When2Meet. They don't need to link to anything yet.

### LINKED ISSUE #44: [Improvement] [FE] Account Popover Edits
Currently, our account popover looks like this:
Image

1. Grab user data from database to display. Weekly hours can be hardcoded still since we haven't implemented the feature yet.
2. Edit the account information update form to reflect these changes. The functionality for an uneditable email is great, you can keep that! Just change the placeholder boxes and color of the titles.
Image

3. When saving changes, connect to the database and update the user information such that it displays on our account popover.

Figma Link: https://www.figma.com/team_invite/redeem/m7USct7pyXUSdm1R6BbS9B

### PR DESCRIPTION
**Pull request by stevenguyukai**
# 1. Added `My summary` button at the account popover
image

# 2. Member summary page
If the `role` is admin then click the button will route to the RA summary page. If not then it will route to the member's summary page.
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -33,6 +34,7 @@ function App() {
           <Route path='/:orgName' element={<Organization />} />
           <Route path="/folder" element={<Folder/>}/>
           <Route path="/ra-summary" element={<RASummary/>}/>
+          <Route path="/members" element={<Members/>}/>
```
Comment (jessieh9) [inline src/App.js]: Can you rename this to something more specific? We have a page called Members that will be implemented so can you use something else?

File: src/pages/Members.jsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
+import React from "react";
```
Comment (jessieh9) [inline src/pages/Members.jsx]: Same comment, can we name the page differently?

**Review by jessieh9 — CHANGES_REQUESTED** [review]
Looks good, but can you also change the button styling to look more similar to the Figma? More minimalistic and less width. 
image

**Comment by stevenguyukai** [comment]
the button is changed to smaller size
image

and the naming for that module is changed from `members` to `personalSummary`

File: src/components/AccountPopup.jsx [hunk 3]
```diff
@@ -185,18 +184,8 @@ function AccountPopup() {
...
-                            {/*                  backgroundColor={"#e3e5e7"}*/}
-                            {/*                  padding={"5px"}*/}
-                            {/*                  borderRadius={"5px"}*/}
-                            {/*                  marginTop={"5px"}>*/}
-                            {/*            {task}*/}
-                            {/*        </Checkbox>*/}
-                            {/*    ))}*/}
-                            {/*</VStack>*/}
+                            <Box></Box>
+                            <Button onClick={() => userData.role.toLowerCase()==="admin"?navigate("/ra-summary"):navigate("/personal-summary")} width={"40%"} backgroundColor={"white"} border={"black solid 1px"} borderRadius={"10px"} height={9}>My summary</Button>
```
Comment (jessieh9) [inline src/components/AccountPopup.jsx]: Can you fetch this data from the database? We have a table for admins vs members :)

File: src/components/AccountPopup.jsx [hunk 4]
```diff
@@ -30,7 +30,7 @@ const userData = {
     name: "Yukai Gu",
```
Comment (jessieh9) [inline src/components/AccountPopup.jsx]: Can you also grab the name, email, description from the DB?

**Review by jessieh9 — CHANGES_REQUESTED** [review]

**Comment by stevenguyukai** [comment]
# Update
## 1. Connect to Database
Now Name, Status, Bio, and UserType(True=ADMIN, False=Member) will be saved and get from db.

image

## 2. Edit Account
The placeholder box and title style has been changed. And the information will be saved to data base after clicking `SAVE` button.

image

## 3. Success toast
After saving successfully, a toast will show to the user.

image

## 4. Database modify
Add a column in Public.Users called `status` which will store users self-wrote word

image



# Close
This will close issue #44

**Review by jessieh9 — APPROVED** [review]

**Pull request merged**